app/src/views/private/components/comments-sidebar-detail/comment-item.vue

```diff
@@ -12,99 +12,76 @@
 			@cancel="cancelEditing"
 		/>
 
-		<div v-else v-md="activity.display" class="content selectable" />
+		<div v-else v-md.newtab="activity.display" class="content selectable" />
```

Comment:
Not too big of a fan of this modifier style v-md configuration. Is there any better way we can pass options to `v-md`?

Maybe we can update the value itself, and have the directive check for string vs object:

```html
<div v-md="{ value: activity.display, target: '_blank' }" />
```